ChangeLog

```diff
@@ -56,6 +56,7 @@ SMPI:
 Java:
  - Due to an internal bug, Msg.run() must now be your last line.
    We hope to fix it in a future release, and we are sorry for the inconvenience.
+ - Expose host load plugin (i.e. loadInit, getCurrentLoad, getComputedFlops,getAvgLoad)
```

Comment:
Could you please put this stanza in the changelog of the upcoming 3.22, not under the already released 3.21 ? ;)
Yes, that'd be the first documented change in 3.22, but we released last week or so.